.setup/INSTALL_SUBMITTY_HELPER.sh

```diff
@@ -251,18 +251,21 @@ if [ "${WORKER}" == 0 ]; then
     mkdir -p ${SUBMITTY_DATA_DIR}/vcs/git
 fi
 
+
+# ------------------------------------------------------------------------
+# Create the logs directories that exist on both primary & worker machines
 mkdir -p ${SUBMITTY_DATA_DIR}/logs
 mkdir -p ${SUBMITTY_DATA_DIR}/logs/autograding
-mkdir -p ${SUBMITTY_DATA_DIR}/logs/emails
 mkdir -p ${SUBMITTY_DATA_DIR}/logs/autograding/stack_traces
 mkdir -p ${SUBMITTY_DATA_DIR}/logs/bulk_uploads
-
-#Make site logging directories if not in worker mode.
+mkdir -p ${SUBMITTY_DATA_DIR}/logs/emails
```

Comment:
I think both emails and bulk_uploads are only things that happen on the primary machine and should be moved inside the if statement?

Reply:
thanks! changed